File: Assignments/Assignment_21/p1.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows:\n");
        scanf("%d",&row);
        int space=row-1,num=65;
        for(int i=1;i<=row;i++){

                for(int j=1;j<=row;j++){

                        if(j<=space){

                                printf("\t");
                        }else{

                                printf("%c\t",num);
                                num++;
                        }
                }printf("\n\n");
                space--;
        }
}

File: JAVA/12.Space_Triangle/p1.java
import java.util.*;

class Demo{

        public static void main(String[] args){

                Scanner sc = new Scanner(System.in);
                System.out.println("Enter no. of rows:");
                int row=sc.nextInt();

                int space=row-2;

                for(int i=0;i<row;i++){

                        int count =1;
                        for(int j=0;j<row;j++){

                                if(j<=space){

                                        System.out.print("\t");
                                }else{

                                        System.out.print(count+"\t");
                                        count++;
                                }

                        }space--;
                        System.out.println();
                }
        }
}
